abandoned-receivers/listing-05-problem/main.go

```diff
@@ -0,0 +1,62 @@
+// Example program to show a goroutine leak. We start goroutines that range
+// over a channel but nothing ever closes the channel.
+package main
```

Comment:
I really really dislike this example. I don't want anyone writing code like this for any reason. :(

Reply:
@jcbwlkr Check out this refactoring. I think it will help with the post.

https://play.golang.org/p/vdCpAWdI3kv